app/views/user_mailer/statistic.text.slim

```diff
@@ -0,0 +1,3 @@
+' User#statistic
+= "\r\n" * 2
```

Comment:
wtf?

Reply:
I didn't write it, this file generated by 'rails g mailer' command